src/worker/core/player/fuzzRating.ts

```diff
@@ -1,17 +1,52 @@
+import { PHASE } from "../../../common";
 import { g, helpers } from "../../util";
 
-const fuzzRating = (rating: number, fuzz: number): number => {
+// Generate plus minus from ovr
+const getO2PM = (ovr: number) => {
+	const iv = [72.7, 7.28 * 2, 18.1 / 2];
+	const minVal = -2.021381747657189;
+	return minVal + iv[1] / (1 + Math.exp(-(ovr - iv[0]) / iv[2]));
+};
+// get ovr from plus minus
+const getPM2O = (pm: number) => {
+	const iv = [72.7, 7.28 * 2, 18.1 / 2];
+	const minVal = -2.021381747657189;
+	//clip plus minus, don't send players to 0 lol
+	const pmC = Math.min(iv[1] + minVal, Math.max(minVal + 0.11, pm));
+	// invert the sigmoid above
+	const ovr = iv[0] - iv[2] * Math.log(iv[1] / (pmC - minVal) - 1);
+	// clip overall
+	const ovrC = Math.min(100, Math.max(0, ovr));
+
+	return ovrC;
+};
+
+const fuzzRating = (rating: number, fuzz: number, isDraft = false): number => {
```

Comment:
Would need to accept season as a parameter too, right? Otherwise fuzz for past season ratings would be constantly oscillating.

Reply:
It could default to `g.get("season")` since in many places it's only the current season, probably.

Reply:
But you don't store fuzz per season, do you? That seems a little annoying and probably something you'd want to do deterministically on the fly then.